### ISSUE TITLE
Maybe the kata-shim binary was missing in 1.4 new release.

### ISSUE THREAD

**Issue by clarklee92** [kata-containers/runtime#946]
# Description of problem

Did anyone find that kata-shim binary file was missing in new release:"kata-static-1.4.0-x86_64.tar.xz"?
Or you have intention of doing so?

**Comment by grahamwhaley** [comment]
/cc @jcvenegas @bergwolf 
Thanks for reporting @clarklee92

**Comment by bergwolf** [comment]
@clarklee92 Thanks for reporting. I confirm the missing `kata-shim` file in the 1.4.0 release tarball. It turns out that we have modified kata-shim's Makefile and removed the `libexecdir` requirement. But the release script is still adding it. I've uploaded a new tarball of the binaries to the 1.4.0 release page.

I also sent a pr in the release repo to fix the releasing script as well, see github.com/kata-containers/packaging#258
